src/python/pants/notes/1.30.x.rst

```diff
@@ -5,6 +5,150 @@ This document describes releases leading up to the ``1.30.x`` ``stable`` series.
 
 See https://pants.readme.io/v1.30/docs/release-notes-1-30 for an overview of the changes in this release.
 
+1.30.0rc0 (6/19/2020)
+-----------------------
```

Comment:
The renderer might complain about the line length not matching the underscore line here.